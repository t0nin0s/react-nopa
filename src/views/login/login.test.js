import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Login from './login'

test('Login component renders as expected', () => {
  const component = shallow(<Login />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
