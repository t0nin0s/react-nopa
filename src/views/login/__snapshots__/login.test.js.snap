// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component renders as expected 1`] = `
<Layout>
  <div
    className="content"
  >
    <section
      className="section section--primary"
    >
      <div
        className="container centered"
      >
        <h1
          className="section__h1 section__h1--margin"
        >
          Which bank does this account belong to?
        </h1>
        <p
          className="section__p"
        >
          Track of all your payments by connecting as
                        many bank accounts as you'd like to your Nopa
                        account and get updates your balance instantly.
                        Plus it's free.
        </p>
        <LoginForm />
      </div>
    </section>
  </div>
</Layout>
`;
